yktracec/src/jitmodbuilder.cc

```diff
@@ -1293,7 +1322,13 @@ class JITModBuilder {
           continue;
         }
 
-        if (Idx > 0 && Idx < InpTrace.Length() - 1) {
+        size_t LastBlockIndex;
```

Comment:
Assuming https://github.com/ykjit/yk/pull/959 merges, this chunk of code can be removed.

Reply:
Removed on merge with master 👉 https://github.com/ykjit/yk/pull/939/commits/d05ddbfdd10504184b897a7bc440bae7e410c436